scripts/exgdas_enkf_update.sh

```diff
@@ -96,11 +101,11 @@ if [ $SUFFIX = ".nc" ]; then
    use_gfs_ncio=".true."
    use_gfs_nemsio=".false."
    paranc=${paranc:-".true."}
+   WRITE_INCR_ZERO="incvars_to_zero= $INCREMENTS_TO_ZERO,"
```

Comment:
will moving this out of the `if-else` block zero out increments when `DO_CALC_INCREMENT=YES` as well?

Reply:
Yes, this will zero out the defined increments as well when DO_CALC_INCREMENT=YES. When I ran tests with this variable in the if statement as original, the DO_CALC_INCREMENT=YES version would break as this variable was needed but wasn't defined in the positive side of the if statement. I ran a test with this variable as is or empty and didn't notice a change in the early cycle analysis file output. If there is a belief that this could cause issues with other applications I can move this back into the if statement and create a positive side variable as an empty string.

Reply:
no. this is fine.  I am sure @CatherineThomas-NOAA will correct us if she thinks differently in the review.

Reply:
Moving $WRITE_INCR_ZERO outside the block doesn't zero the increments for the DO_CALC_INCREMENT=YES case, but it doesn't hurt it.  WRITE_INCR_ZERO is only used in the writeincrement routine in the EnKF, so it gets ignored in the case where the analysis is written instead.  

For DO_CALC_INCREMENT=YES, the increment is computed during ecen and the zeroing out is controlled by $INCREMENTS_TO_ZERO.  In Travis’s tests, the cloud increments in the analysis file remain zero because we don’t currently have cloud control variables in the EnKF, so their increments will be zero no matter what.  

This brings up another point though that we need to consider.  If there are variables we want to zero out for the GEFS, we will need to do additional work.